### PR TITLE
Added de_DE.lang

### PR DESCRIPTION
**Pull request by Soro300**
Translated some of the old .lang file to German. I know you're German
too Blu but I thought why not ;)

### CODE REVIEW

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,312 @@
+﻿# BLOCKS
+tile.ImmersiveEngineering.ore.Copper.name=Kupfer Erz
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: Has to be `Kupfererz`. Same for all the other ores.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.ore.Nickel.name=Nickel Erz
+tile.ImmersiveEngineering.storage.Copper.name=Kupferblock
+tile.ImmersiveEngineering.storage.Aluminum.name=Aluminiumblock
+tile.ImmersiveEngineering.storage.Lead.name=Bleiblock
+tile.ImmersiveEngineering.storage.Silver.name=Silberblock
+tile.ImmersiveEngineering.storage.Nickel.name=Nickelblock
+tile.ImmersiveEngineering.storage.Constantan.name=Konstantanblock
+tile.ImmersiveEngineering.storage.Electrum.name=Elektrumblock
+tile.ImmersiveEngineering.storage.Steel.name=Stahlblock
+tile.ImmersiveEngineering.storage.CoilCopper.name=Kupfer Draht Spule
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: No spaces needed here either.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.metalDevice.relayHV.name=Hochspannungs Kabel Überbrücker
+tile.ImmersiveEngineering.metalDevice.connectorHV.name=Hochspannungs Kabel Verbinder
+tile.ImmersiveEngineering.metalDevice.capacitorHV.name=Hochspannungs Kondensator
+tile.ImmersiveEngineering.metalDevice.transformerHV.name=Hochspannungs Transformator
+tile.ImmersiveEngineering.metalDevice.dynamo.name=Kraft Generator
+tile.ImmersiveEngineering.metalDevice.thermoelectricGen.name=Thermoelektrischer Generator
+tile.ImmersiveEngineering.metalDevice.conveyorBelt.name=Fließband
+tile.ImmersiveEngineering.metalDevice.furnaceHeater.name=Externer Wärmer
+
+tile.ImmersiveEngineering.metalMultiblock.lightningRod.name=Blitz Stab Basis
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: "Blitz Stab"? Seriously? Sounds like you translated word-by-word.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.metalDecoration.lightEngineering.name=Leichter Ingenieursblock
+tile.ImmersiveEngineering.metalDecoration.connectorStructural.name=Struktureller Kabel Konektor
+
+tile.ImmersiveEngineering.woodenDevice.post.name=Holz Mast
+tile.ImmersiveEngineering.woodenDevice.watermill.name=Wassermühle
+tile.ImmersiveEngineering.woodenDevice.windmill.name=Windmühle
+tile.ImmersiveEngineering.woodenDevice.windmillAdvanced.name=Verbesserte Windmühle
+tile.ImmersiveEngineering.woodenDevice.crate.name=Lagerungs Kiste
+
+tile.ImmersiveEngineering.woodenDecoration.treatedWood.name=Behandelte Holz Planken
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: "Planks" are translated with "Bretter" in Minecraft.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+item.ImmersiveEngineering.bullet.armorPiercing.name=Rüstungsdurchbrechende Ladung
+item.ImmersiveEngineering.bullet.buckshot.name=Schrot Ladung
+item.ImmersiveEngineering.bullet.HE.name=High-Explosive Ladung
+item.ImmersiveEngineering.bullet.dragonsbreath.name=Drachenatem Ladung
+
+#CREATIVE
+itemGroup.ImmersiveEngineering=Immersive Engineering
+
+#DESCRIPTIONS
+desc.ImmersiveEngineering.info.attachedTo=Verbindet von %1$s, %2$s, %3$s
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: Are you sure you translated this yourself? This sounds like word-by-word google translate...
Should have been `Verbunden mit`

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+chat.ImmersiveEngineering.warning.wrongCable=Du kannst hier kein Kabel anschließen
+chat.ImmersiveEngineering.warning.wrongDimension=Du bist in der falschen Dimension
+chat.ImmersiveEngineering.warning.sameConnection=Du kannst kein Kabel am selben Ort anschließen
+chat.ImmersiveEngineering.warning.tooFar=Du bist zu weit entfernt von der vorhrigen Verbindung
+chat.ImmersiveEngineering.warning.cantSee=Die Verbindung ist versperrt
+chat.ImmersiveEngineering.info.averageLoss=Durchschnittliche Verlust Rate zwischen den Punkten: %1$s RF
+chat.ImmersiveEngineering.info.energyStorage=Energie gespeichert: %1$s/%2$s RF
+
+#RECIPES
+recipe.ImmersiveEngineering.hammerCrushing=Zerbrechen
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: I'd say "Zermahlen" or "Zerstoßen"

**Comment by Vexatos** [comment]
Next time, please consider
1. Not using Google translate (if you haven't, you are not a native speaker)
2. Not translating word-by-word (why would you ever do so?)
3. Not using spaces everywhere. Roughly half of this file is wrong simply due to too many free spaces.

**Comment by Soro300** [comment]
Yeah , sorry if I dissappoint you but 1. I'm an actual native German Speaker and didn't usw Google Translate. 2. Some of the things weren't even implemented so I didn't have a Chance to know what they die so I translated it linke I would think 3. The spaces are dumb I have to damit
I'm NOT a native speaker in English so there's that. It was intended as a first approach to a translation and I'm just not experienced as you with your hundreds of translations.
BTW Sorry but you really sound rude over inexperienced translators.

**Comment by Vexatos** [comment]
Well, I know no German person who'd actually use this many spaces anywhere. For things like "Lightning Rod", you should try using [some proper dictionary](http://de.pons.com/).

**Comment by Soro300** [comment]
I updated now I hope most of the Spaces are right. Why I did them is because I didn't knew how they have it in vanilla minecraft. And the spaces were pretty dumb. The "Lightning Rod" was completly wrong because I didn't knew how to translate it and translated it word by word( Sorry). And I used [this] (http://www.dict.cc/) as an dictionary.

**Comment by Vexatos** [comment]
> because I didn't knew how they have it in vanilla minecraft

You shouldn't try translating a mod when you aren't playing in the language you're translating to.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.ore.Nickel.name=Nickelerz
+tile.ImmersiveEngineering.storage.Copper.name=Kupferblock
+tile.ImmersiveEngineering.storage.Aluminum.name=Aluminiumblock
+tile.ImmersiveEngineering.storage.Lead.name=Bleiblock
+tile.ImmersiveEngineering.storage.Silver.name=Silberblock
+tile.ImmersiveEngineering.storage.Nickel.name=Nickelblock
+tile.ImmersiveEngineering.storage.Constantan.name=Konstantanblock
+tile.ImmersiveEngineering.storage.Electrum.name=Elektrumblock
+tile.ImmersiveEngineering.storage.Steel.name=Stahlblock
+tile.ImmersiveEngineering.storage.CoilCopper.name=Kupferdraht Spule
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: Needs to be `Kupferdrahtspule`. Same for the other coils.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.storage.CoilElectrum.name=Elektrumdraht Spule
+tile.ImmersiveEngineering.storage.CoilHV.name=Stark-Stromdraht Spule
+
+tile.ImmersiveEngineering.metalDevice.connectorLV.name=Niedrigspannungs Kabel Verbinder
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: Still way too many spaces.

File: src/main/resources/assets/immersiveengineering/lang/de_DE.lang [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+tile.ImmersiveEngineering.metalDevice.connectorHV.name=Hochspannungs Kabel Verbinder
+tile.ImmersiveEngineering.metalDevice.capacitorHV.name=Hochspannungs Kondensator
+tile.ImmersiveEngineering.metalDevice.transformerHV.name=Hochspannungs Transformator
+tile.ImmersiveEngineering.metalDevice.dynamo.name=Kraft Generator
+tile.ImmersiveEngineering.metalDevice.thermoelectricGen.name=Thermoelektrischer Generator
+tile.ImmersiveEngineering.metalDevice.conveyorBelt.name=Fließband
+tile.ImmersiveEngineering.metalDevice.furnaceHeater.name=Ofen Heizer
+
+tile.ImmersiveEngineering.metalMultiblock.lightningRod.name=Blitzableiter
+tile.ImmersiveEngineering.metalMultiblock.dieselGenerator.name=Diesel Generator
```
Comment (Vexatos) [inline src/main/resources/assets/immersiveengineering/lang/de_DE.lang]: `Dieselgenerator`

**Comment by Vexatos** [comment]
It appears you only fixed the ingots, dusts and blocks...

**Comment by Soro300** [comment]
You know what I will just close this pr and don't bother you anymore. Do it yourself. Just wanted to help.

**Comment by Vexatos** [comment]
I appreciate you wanting to help, but I'd rather have no English translation at all in a mod than one with wrong grammar, as the former is much more common and thus easier to accept.

**Comment by Soro300** [comment]
I understand that completly and I like a right translation much more too it is just the thing that I usually don't think about spaces when speaking and normally when I write I do it right. I think it is because in a translation the words are out of context and I seperate everything. Sorry ;) I will maybe come back and look if I can improve the translation like with Thaumcraft.

**Comment by Vexatos** [comment]
Well, translating everything out of any context is indeed one of the hardest challenges.

**Pull request closed without merging**